apps/backend/src/main/java/ch/heig/pdg/backend/presentation/InventoryController.java

```diff
@@ -44,8 +46,10 @@ public ResponseEntity<InventoryDTO> deleteInventory(Integer id) {
     @Override
     public ResponseEntity<List<InventoryDTO>> getInventories() {
         HugoSearchFilter<Inventory> filter = HugoSearchFilter.build(this.httpServletRequest, Inventory.class);
+        List<InventoryDTO> inventories = this.inventoryService.getInventories(filter);
         return new ResponseEntity<>(
-                this.inventoryService.getInventories(filter),
+                inventories,
+                new LinkedMultiValueMap<>(Map.of("X-Total", List.of(String.format("%d", inventories.size())))),
```

Comment:
Wait!
Does'nt this just return the size of the data already returned?

I think there's been a misunderstanding.
When doing `/items?where[id][$gt]=2&limit=5&offset=0`, it should return _5 items_ and the total being 13 (as an example).

So the frontend knows that, for the given filter, there 13 items in total (8 more), so it can make `/items?where[id][$gt]=2&limit=5&offset=5` when it needs to load more.

Reply:
So in essence what you want is the total of entities that would be returned if you did not put the limit/offset filter ?